### ISSUE TITLE
Is it possible to change the animation type for the transition between last and first slides?

### ISSUE THREAD

**Issue by sarah97302** [chrishumboldt/Flickerplate#41]
I'm using arrow-constraints="false" to allow looping back to first slide from last slide. No matter how i set the flick-animation type, I am getting a different animation type between the last and first slides -- when you click the right arrow on the last slide it shuffles through all of the slides very quickly before it returns to the first slide. I want a smooth transition from last to first, that looks the same as all the other transitions. Example here: http://03be309.netsolhost.com/wordpress/

Is there a way to change this setting, something I can modify in the .js file? I can't figure out where/how to do this.

**Comment by Jansbury** [comment]
I agree. Even just a loop, so it's continuous with no sudden reset. Someone please help!

**Comment by AlexandruDoda** [comment]
:+1: I agree
